selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -132,6 +146,7 @@ public CodenvyEditor(
     String CONTEXT_MENU = "//div[@id='menu-lock-layer-id']/div[2]";
     String EDITOR_TABS_PANEL = "gwt-debug-multiSplitPanel-tabsPanel";
     String ACTIVE_LINE_NUMBER = "gwt-debug-cursorPosition";
+    String TAB_CONTEXT_MENU_BODY = "//*[@id='gwt-debug-contextMenu/closeAllEditors']/parent::tbody";
```

Comment:
To complex xpath, IMHO. It's better to address the element by id, or name.

Reply:
I meant `parent::tbody` part. It's not reliable to address parent element so as there could be anything instead of required one, isn't it?

Reply:
In this case it is quite acceptable.